src/features/people/ActorPage/About/index.js

```diff
@@ -22,11 +23,13 @@ const About = () => {
         <Name>{actorInfo.name}</Name>
         <BirthInfo>
           <div>
-            <SpecificInfo>Date of birth:</SpecificInfo>
-            {actorInfo.birthday}
+            <SpecificInfo>
+              <OptionalInfo>date of</OptionalInfo> birth:
```

Comment:
` </OptionalInfo> birth: ` is that `birth:` out for a reason ?

Reply:
In figma there is only birth on mobile view